app/Modules/Media/database/migrations/2018_03_31_104532_create_media_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
             $table->string('name')->nullable();
             $table->boolean('is_public')->default(false);
             $table->integer('owner_id')->nullable();
+            $table->string('meta')->nullable();
```

Comment:
```suggestion
            $table->json('meta')->nullable();
```

Reply:
I changed type to jsonb. Actually I made the same trick which we are using for the `value` field at the `Settings`